lib/order.rb

```diff
@@ -0,0 +1,58 @@
+class Order
+
+  attr_reader :dishes, :total, :calc, :menu
+
+  def initialize(dish_class = Dish)
+    @dish_class = dish_class
+    @dishes = []
+    @dish = nil
+    @total = 0
```

Comment:
Similar to my comment below, it is better to avoid duplicating state. What I mean by this is that `total` is the sum of all the dishes in the order, so it is better to calculate that when needed, than store it separately in its own variable. Instead, it is better to create a total method which calculates the total from the stored dishes.